google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -207,8 +207,11 @@ public static Builder newBuilder(String subscription, MessageReceiver receiver)
   }
 
   /** Returns the delivery attempt count for a received {@link PubsubMessage} */
-  public static int getDeliveryAttempt(PubsubMessage message) {
-    return Integer.parseInt(message.getAttributesOrDefault("googclient_deliveryattempt", "0"));
+  public static Integer getDeliveryAttempt(PubsubMessage message) {
+    if (!message.containsAttributes("googclient_deliveryattempt")) {
```

Comment:
Should we add a unit test for this behavior?

Reply:
Yes, good idea. Unit test added